daml-foundations/daml-tools/daml-extension/syntaxes/daml12.tmLanguage.xml

```diff
@@ -46,6 +46,18 @@
         <key>name</key>
         <string>constant.language.empty-list.daml</string>
       </dict>
+      <dict>
+        <key>match</key>
+        <string>\b(choice)\b</string>
```

Comment:
Why the brackets here? What do they mean?

Reply:
Nope not needed I was experimenting with something,